.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Installing Dependencies
-        run: npm ci
+        run: npm install
```

Comment:
Perhaps we could have the `--ignore-scripts` here?

Reply:
Makes sense.

Reply:
If this is preferred I'd put it into `.npmrc` instead.